it/src/test/java/org/eclipse/jkube/integrationtests/springboot/zeroconfigfatjar/ZeroConfigFatJar.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.integrationtests.springboot.zeroconfigfatjar;
+
+import io.fabric8.junit.jupiter.api.KubernetesTest;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import org.eclipse.jkube.integrationtests.JKubeCase;
+import org.eclipse.jkube.integrationtests.maven.MavenCase;
+
+import static org.eclipse.jkube.integrationtests.assertions.PodAssertion.awaitPod;
+import static org.eclipse.jkube.integrationtests.assertions.ServiceAssertion.awaitService;
+import static org.hamcrest.Matchers.hasSize;
+
+@KubernetesTest(createEphemeralNamespace = false)
+public class ZeroConfigFatJar implements JKubeCase, MavenCase {
```

Comment:
I'm assuming this is because Spring is less than x version. right?

Reply:
You're right. I've updated PR description